openid-4-verifiable-presentations-1_0.md

```diff
@@ -2198,8 +2223,7 @@ ISO/IEC TS 18013-7 Annex B [@ISO.18013-7] and ISO/IEC 23220-4 [@ISO.23220-4] Ann
 * Rules for the `presentation_definition` Authorization Request parameter.
 * Rules for the `presentation_submission` Authorization Response parameter.
 * Wallet invocation using the `mdoc-openid4vp://` custom URI scheme.
-* Defines the OpenID4VP-specific `Handover` CBOR structure and how OpenID4VP Authorization Request and Request Object parameters apply to the `SessionTranscript` CBOR structure and `DeviceResponse` CBOR structure as specified in ISO/IEC 18013-5 [@ISO.18013-5] and ISO/IEC 23220-4 [@ISO.23220-4].
-* Required Wallet and Verifier Metadata parameters and their values.
+* Required Wallet and Verifier Metadata parameters and their values when OpenID4VP is used with the `mdoc-oid4vp://` custom URI scheme.
```

Comment:
```suggestion
* Required Wallet and Verifier Metadata parameters and their values.
* The `SessionTranscript` and `Handover` CBOR structure when OpenID4VP is used with the `mdoc-oid4vp://` custom URI scheme. Also see (#non-dc-api-invocation).
```

18013-7 does define `SessionTranscript` and `Handover` CBOR structure when OpenID4VP is used with the `mdoc-oid4vp://` custom URI scheme, so I would still mention it

Reply:
That is fine but let's make the other item not `mdoc-oid4vp://` specific.

```suggestion
* Required Wallet and Verifier Metadata parameters and their values when OpenID4VP is used with the `mdoc-oid4vp://` custom URI scheme.
The `SessionTranscript` and `Handover` CBOR structure when the invocation does not use the DC API. Also see (#non-dc-api-invocation).
```

Reply:
Does that work for you @Sakurann ?

Reply:
Committed my suggestion as agreed on the DCP call.